expenses-manager-backend/src/main/java/com/endava/expensesmanager/repository/ExpenseRepository.java

```diff
@@ -12,4 +14,11 @@ public interface ExpenseRepository extends JpaRepository<Expense, Integer> {
     List<Expense> findByUserId(int userId);
 
     List<Expense> findAllByUserIdAndDateBetween(int userId, LocalDateTime startDate, LocalDateTime endDate);
+
+    @Query("SELECT e FROM Expense e WHERE e.user.id = :userId " +
```

Comment:
Try to format SQL query like this 
SELECT
FROM
WHERE

instead of SELECT FROM WHERE
https://medium.com/@BrandonSouthern/sql-best-practices-e1c61e96ee27